### ISSUE TITLE
examples

### ISSUE THREAD

**Issue by ypriverol** [BD2K-DDI/gpmdb-ddi-reader#2]
@felipe great work.. Im out of the office that means the testing in our side will need to way a couple of days.. in anycase im still downloading.. onething I can do is test the final xmls of your tool.. then:
1 can yo share with me a couple of examples to test then in my side.

Regards
Yasset

**Comment by Leprevost** [comment]
thanks @ypriverol. The first proto type results  are located here:
https://github.com/BD2K-DDI/gpmdb-ddi-reader/tree/master/data/records

There are the xml files generated from folder 451.

The file generation is working but we have to check the fields, some of them are blank.

**Comment by ypriverol** [comment]
@Leprevost My comments here after the first iteration:
 - [ ] the id can be the first model in the list of models for each project, for example GPM45100005450. If 
   you go to gpmdb you will find that of this description 14 diffetent models can be found. The id can be 
   the first of them. 
 - [ ] Then the title can be the information that you put right now in the id.
 - [ ] For this model the cell type can be annotated CELL cell type: CL:0000137  osteocyte
 - [ ] The description field should be the comment field in the model, in this case: 
        [Massive](http://proteomics.ucsd.edu/ProteoSAFe/status.jsp?task=TRANCHE-MSV000074112)  Data file: B06-9360_Pr_OBSlong_c. Shotgun proteomics on the osteocytic cell line MLO-Y4, to identify new targets of the anti-osteoporosis drugs Bisphosphonates. MLO-Y4 cells were treated with the Bisphosphonate Risedronate at the concentration 10-7M for 1h and 48h. At the end of the treatment protein extracts were prepared and analysed. The proteome at the basal, untreated condition was used as a reference to whom the proteome from stimulated cells was compared to.

  - [ ] You need to provide this fields:
          <field name="omics_type">Proteomics</field>
          <field name="repository">GPMDB</field>
          <field name="instrument_platform">Instrument</field>
      

  - [ ]  Please see that the massive reference or PX accession can apear in the original description. In 
         those cases you should add those references as follow:
          <ref dbkey="id-dataset-massive" dbname="massive"/> 

 -

**Comment by Leprevost** [comment]
Hey @ypriverol Yasset. Just resolved 4 of the 6 items mentioned above. I need a example of the following cases:

 Then the title can be the information that you put right now in the id.
 For this model the cell type can be annotated CELL cell type: CL:0000137 osteocyte

I'm note sure how to create those tags.

**Comment by ypriverol** [comment]
@Leprevost for the cell type add the name of the cell-type only osteocyte. like:
   ```xml
    <field name="cell_type">osteocyte</field>
   ```

**Comment by ypriverol** [comment]
@Leprevost if the information is not provided or not available please add the corresponding tag with not available:
  ```xml
       <field name="cell_type">Not available</field>
        <field name="disease">Not available</field>
        <field name="tissue">Not available</field>
  ```

**Comment by Leprevost** [comment]
thanks, updated!

**Comment by ypriverol** [comment]
Projects? Can I see a couple of them? Can you make them available to have a look?

**Comment by Leprevost** [comment]
sure, check here:
https://github.com/BD2K-DDI/gpmdb-ddi-reader/tree/master/data/xml

**Comment by ypriverol** [comment]
@Leprevost I was taking a look to the [GPM10100701083] (https://raw.githubusercontent.com/BD2K-DDI/gpmdb-ddi-reader/d8fb6be2a4b5ecaeb2e9124a6e8dc47101f08b56/data/xml/GPMDB_EBE_484.xml)  in the [gpmdb entry](GPM10100701083) apparently the project has some metadata associated like: 

--Project
-- Project Name

But in your file this information is missing.  

Another small bug, try to do not use this extandard for the empty tags:

</cross_references>

**Comment by Leprevost** [comment]
yes, we are not there yet, but slowly progressing. I'm just showing you the
changes from my updates. I think that different model files for the same
project have different information. For example, if a project has 10
models, some of them can have the project name and some don't.

What you mean by extandard on empty tags?

What sould I put on the <name> tags?

**Comment by ypriverol** [comment]
when you have empty tags for best practices I would like to have:
```xml
<cross_references></cross_references>
```

rather than:

```xml
</cross_references>
```

**Comment by ypriverol** [comment]
@Leprevost Can we add to each entry the date of running if is provided.

**Comment by Leprevost** [comment]
sure, i will work on that. And what about the <name> tags?

**Comment by Leprevost** [comment]
I fixed that cross_reference tag issue, fixed some details and generated a new set of xml files.
